scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -142,12 +144,13 @@ private static String getFrom() throws JobEmailNotificationException {
         return from;
     }
 
-    private String getTo() throws JobEmailNotificationException {
+    private String[] getTo() throws JobEmailNotificationException {
```

Comment:
Why using an array and not a List?

Reply:
I think that hiding the array directly after the split by creating a List from the array would make the code more resilient and make it more usable.